app/qml/main.qml

```diff
@@ -715,6 +715,21 @@ ApplicationWindow {
     onManageAccountClicked: Qt.openUrlExternally(__inputHelp.merginSubscriptionLink)
   }
 
+  MMMonthlyContributorsLimitDialog {
+    id: monthlyContributorsLimitDialog
+
+    apiSupportsSubscription: __merginApi.apiSupportsSubscriptions
```

Comment:
We do not have this limit in other servers apart from Cloud, so this info is not needed
```suggestion

```

Reply:
Removed the need for support subscription and link to pricing page